src/misc/appCreator.ts

```diff
@@ -47,10 +47,15 @@ export class ${ pascalCase(this.fd.info.name) }App extends App {
-
+    private createBlankIcon(imageIconPath: string): void {
+        let base64String = '';
+        if (imageIconPath) {
+            const bitmap = fs.readFileSync(imageIconPath);
```

Comment:
Does this have any chance of throwing errors while when attempting to read it the file?

Reply:
It can only show errors if imageIconPath is invalid or empty, and as I have already checked for that while entering the path and in the if statement respectively, there should be no problem.